sdk/examples/wallet/15_destroy_foundry.rs

```diff
@@ -13,39 +21,43 @@ async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
-    // Create the wallet
-    let wallet = Wallet::builder().finish().await?;
+    // Access the wallet we generated with `--example create_wallet`
+    let wallet = Wallet::builder()
+        .with_storage_path(&var("WALLET_DB_PATH").unwrap())
+        .finish()
+        .await?;
+    let account = wallet.get_account(&var("ACCOUNT_ALIAS_1").unwrap()).await?;
 
-    // Get the account we generated with `01_create_wallet`
-    let account = wallet.get_account("Alice").await?;
     // May want to ensure the account is synced before sending a transaction.
     let balance = account.sync(None).await?;
 
-    // Get the first foundry
+    // We try to destroy the first foundry in the account
     if let Some(foundry_id) = balance.foundries().first() {
```

Comment:
Should we find a foundry that can be destroyed instead? Most likely the first one still has tokens

Reply:
I would rather keep the example simple. We could mention the prerequisites of this example to actually work by mentioning it in the doc comments like we do with other examples... so ... make sure to burn all tokens with example `burn_native_token` example. Happy to here some suggestions. Finding a proper foundry id like you suggest is also possible, but there's no guarantee that's work either, so I kind of prefer just docs. Wdyt? Any suggestions?

Reply:
yea ive realized finding one is more annoying. Ive made a related issue: https://github.com/iotaledger/iota-sdk/issues/524